renormalizer/mps/tests/test_mpdm.py

```diff
@@ -43,38 +43,51 @@ def test_from_mps():
 @pytest.mark.parametrize(
     "mol_list, etot_std, occ_std, rtol",
     (
-        [
-            parameter.hybrid_mol_list,
-            0.0853441664951,
-            [0.20881609, 0.35239430, 0.43878960],
-            5e-3,
-        ],
+        # explicit hybrid algorithm will be removed
```

Comment:
So the tests for the hybrid scheme currently do not work?

Reply:
It works. I skip it to save time.